setup.cfg

```diff
@@ -103,37 +100,45 @@ github_project = lcogt/banzai
 version = 1.0.6
 
 [options]
-setup_requires =
-    cython
-    numpy==1.17.4
+zip_safe = False
+packages = find:
 install_requires =
     astropy>=3.0
     scipy
     sqlalchemy>=1.3.0b1
     logutils
-    numpy==1.17.4
+    numpy
     cython
     mysql-connector-python
     lcogt_logging==0.3.2
     sep
     kombu==4.4.0
-    amqp==2.4.2
+    amqp==2.6.0
```

Comment:
Upgraded due to a transitive dependency issue with celery/kombu https://github.com/celery/py-amqp/issues/340